### PR TITLE
[Object] Bigpo object documentation (+ some gameplay_keep)

### PR DESCRIPTION
**Pull request by isghj5**
Before opening this PR, ensure the following:
- `./format.sh` was run to apply standard formatting.
- `make` successfully builds a matching ROM.
- No new compiler warnings were introduced during the build process.
    - Can be verified locally by running `tools/warnings_count/check_new_warnings.sh`
- New variables & functions should follow standard naming conventions.
- Comments and variables have correct spelling.
---

First object documentation attempt

### CODE REVIEW

File: src/overlays/actors/ovl_Obj_Aqua/z_obj_aqua.c [hunk 1]
```diff
@@ -66,7 +66,7 @@ Vec3f D_80ACC308 = { 1.0f / 1000.0f, 7.0f / 10000.0f, 1.0f / 1000.0f };
 Vec3f D_80ACC314 = { 8.6f / 1000.0f, 8.0f / 10000.0f, 8.6f / 1000.0f };
 Vec3f D_80ACC320 = { 1.0f / 100.0f, 2.6f / 1000.0f, 1.0f / 100.0f };
 
-extern Gfx D_0407D590[];
+extern Gfx gGameplayKeepDrawFlameDL[];
```
Comment (AngheloAlf) [inline src/overlays/actors/ovl_Obj_Aqua/z_obj_aqua.c:69]: I know it wasn't you, but this should be removed and the gameplay_keep header should be used instead

File: assets/xml/objects/object_bigpo.xml [hunk 2]
```diff
@@ -1,46 +1,46 @@
...
+        <Texture Name="gBigpoLanternGlassTex" OutName="bigpo_lantern_glass" Format="rgba16" Width="16" Height="16" Offset="0x1E70" />
+        <Texture Name="gBigpoTornRobeTex" OutName="bigpo_torn_robe" Format="rgba16" Width="32" Height="32" Offset="0x2070" />
+        <Texture Name="gBigpoArmSectionTex" OutName="bigpo_arm_section" Format="rgba16" Width="16" Height="16" Offset="0x2870" />
+        <Texture Name="gBigpoHoodEdge" OutName="bigpo_hood_edge" Format="rgba16" Width="8" Height="8" Offset="0x2A70" />
+        <Texture Name="gBigpoNecklaceBeadsTex" OutName="bigpo_necklace_beads" Format="rgba16" Width="16" Height="8" Offset="0x2AF0" />
+        <Texture Name="gBigpoShawlTex" OutName="bigpo_shawl" Format="rgba16" Width="16" Height="16" Offset="0x2BF0" />
+        <Texture Name="gBigpoGlowingEyeTex" OutName="bigpo_glowing_eye" Format="rgba16" Width="16" Height="16" Offset="0x2DF0" />
         <!-- <Blob Name="object_bigpo_Blob_003200" Size="0x350" Offset="0x3200" /> -->
         <!-- <Blob Name="object_bigpo_Blob_003790" Size="0x370" Offset="0x3790" /> -->
```
Comment (AngheloAlf) [inline assets/xml/objects/object_bigpo.xml:20]: Are those blobs completely unused? :o

**Review by AngheloAlf — COMMENTED** [review]
Nice work!
Just some minor comments

File: assets/xml/objects/object_bigpo.xml (hunk 2 above)
Comment (isghj5) [reply assets/xml/objects/object_bigpo.xml:20]: No idea. I haven't walked through every display list to check if they use it, they aren't reference in the actor code, and I don't know enough about this subject to be able to analyze the binary directly.

They were commented out like that when I started.

File: src/overlays/actors/ovl_En_Bigpo/z_en_bigpo.c [hunk 3]
```diff
@@ -254,6 +243,7 @@ void EnBigpo_Destroy(Actor* thisx, GlobalContext* globalCtx) {
     }
 }
 
+// how did I not name this function???
 void func_80B61914(EnBigpo* this) {
```
Comment (hensldm) [inline src/overlays/actors/ovl_En_Bigpo/z_en_bigpo.c:247]: Are you able to name it now?

File: src/overlays/actors/ovl_En_Bigpo/z_en_bigpo.h [hunk 4]
```diff
@@ -2,6 +2,8 @@
 #define Z_EN_BIGPO_H
 
 #include "global.h"
+#include "objects/object_bigpo/object_bigpo.h"
+#include "objects/gameplay_keep/gameplay_keep.h"
```
Comment (EllipticEllipsis) [inline src/overlays/actors/ovl_En_Bigpo/z_en_bigpo.h:6]: These should go in the C file if they're only needed there.

File: src/overlays/actors/ovl_En_Dno/z_en_dno.h [hunk 5]
```diff
@@ -2,6 +2,7 @@
 #define Z_EN_DNO_H
 
 #include "global.h"
+#include "objects/gameplay_keep/gameplay_keep.h"
```
Comment (hensldm) [inline src/overlays/actors/ovl_En_Dno/z_en_dno.h:5]: Same here `.c` file.

File: src/overlays/actors/ovl_En_Po_Sisters/z_en_po_sisters.h [hunk 6]
```diff
@@ -2,6 +2,7 @@
 #define Z_EN_PO_SISTERS_H
 
 #include "global.h"
+#include "objects/gameplay_keep/gameplay_keep.h"
```
Comment (hensldm) [inline src/overlays/actors/ovl_En_Po_Sisters/z_en_po_sisters.h:5]: Same here `.c` file

File: src/overlays/actors/ovl_Obj_Aqua/z_obj_aqua.h [hunk 7]
```diff
@@ -2,6 +2,7 @@
 #define Z_OBJ_AQUA_H
 
 #include "global.h"
+#include "objects/gameplay_keep/gameplay_keep.h"
```
Comment (hensldm) [inline src/overlays/actors/ovl_Obj_Aqua/z_obj_aqua.h:5]: Same here `.c` file

File: src/overlays/actors/ovl_Obj_Syokudai/z_obj_syokudai.h [hunk 8]
```diff
@@ -2,6 +2,7 @@
 #define Z_OBJ_SYOKUDAI_H
 
 #include "global.h"
+#include "objects/gameplay_keep/gameplay_keep.h"
```
Comment (hensldm) [inline src/overlays/actors/ovl_Obj_Syokudai/z_obj_syokudai.h:5]: Same here `.c` file

**Review by hensldm — COMMENTED** [review]
A few things I noticed.

**Review by AngheloAlf — APPROVED** [review]

File: assets/xml/objects/object_bigpo.xml (hunk 2 above)
Comment (hensldm) [reply assets/xml/objects/object_bigpo.xml:20]: Might be a good idea to mark on the sheet that bigpo has unaccounted/unused blobs.

**Review by hensldm — APPROVED** [review]
LGTM

**Review by Kenix3 — APPROVED** [review]

**Pull request merged**